frontend/app/components/wp-fast-table/state/wp-table-metadata.service.ts

```diff
@@ -0,0 +1,46 @@
+import {WorkPackageTableMetadata} from '../wp-table-metadata';
+import {States} from '../../states.service';
+import {opServicesModule} from '../../../angular-modules';
+import {State} from '../../../helpers/reactive-fassade';
+import {WPTableRowSelectionState} from '../wp-table.interfaces';
+
+export class WorkPackageTableMetadataService {
+
+  // The current table metadata state
+  public metadata:State<WorkPackageTableMetadata>;
+
+  constructor(public states: States) {
+    "ngInject";
+    this.metadata = states.table.metadata;
+  }
+
+  /**
+   * Returns whether the given column is contained in the current set
+   * of groupable columns.
+   */
+  public isGroupable(name:string):boolean {
+    return !!_.find(this.current.groupableColumns, (column) => column.name === name);
```

Comment:
Contains works only for properties. While it appears to look in all values for an object, I find that to be less precise